CONTRIBUTING.md

```diff
@@ -3,19 +3,23 @@
 Thank you for your interest in that project, and for taking time to contribute!
 
 ## Create a new role
+
 Run the following command to get your new role ready:
-```Bash
-[laptop]$ make create_new_role ROLE_NAME=my_wonderful_role
-```
+
+~~~Bash
```

Comment:
backtits are valid as well

Reply:
sure, but the linter complains if you use different fencing styles in the doc: https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md048.md